### PR TITLE
feat: LUKSO Client diversity spike

### PR DESCRIPTION
**Pull request by Wolmin**
DEV-11260

This PR provides an initial changes done to the repository in order to provide autonomous LUKSO client statistics.

### CODE REVIEW

File: _data/clients-execution.yml [hunk 1]
```diff
@@ -108,3 +115,4 @@
   lang: C++
   donate: https://etherscan.io/address/0x8BFBB529A9E85fDC4b70A4FCdC0D68Bb298B8816
   opensource: true
+  lukso_supported: false
```
Comment (JeneaVranceanu) [inline _data/clients-execution.yml:119]: Reth client missed? lines 98-107.

**Review by JeneaVranceanu — COMMENTED** [review]
Not really competent here but again just some sanity checks.

File: _config.yml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # Read about configuration here: https://jekyllrb.com/docs/configuration/options/
-title: Client Diversity | Ethereum
-description: Improve Ethereum's resilience and protect the network by using a minority client.
-keywords: ethereum, client, diversity, eth, beacon, chain
+title: Client Diversity | LUKSO
+description: Improve LUKSO's resilience and protect the network by using a minority client.
+keywords: ethereum, lukso, client, diversity, eth, beacon, chain
 url: https://clientdiversity.org
```
Comment (mxmar) [inline _config.yml]: Please change url to LUKSO's domain

File: _scripts/collect_data.py [hunk 3]
```diff
@@ -3,38 +3,36 @@
...
 exit_on_save_error = True
 exit_on_report_error = False
 
-rated_token = os.environ.get("RATED_API_KEY")
-migalabs_token = os.environ.get("MIGALABS_API_KEY")
-google_form_error_report_url = os.environ.get("ERROR_REPORT_ENDPOINT")
+google_form_error_report_url = os.environ.get("")
+
+# URLS
+blockprint_api_addr = "http://34.140.62.68:8000"
```
Comment (mxmar) [inline _scripts/collect_data.py]: Can we move to env? `os.environ.get`

**Review by mxmar — APPROVED** [review]
Please solve comments :+1:

**Pull request merged**